Add bootstrap to hooks documentation (#16)
knuckleswtf/scribe-docs: laravel/50-hooks.md

Always wrap these method calls in an `if (class_exists(\Knuckles\Scribe\Scribe::class))` statement. That way, you can push this code to production safely, even if Scribe is installed in dev only.
:::

## `bootstrap()`

> `bootstrap()` allows you to run some code before the generate command being executed. You can use this to update service container bindings, inspect the command instance properties or whatever you wish,

Reviewer: ```suggestion
`bootstrap()` allows you to run some code when the generate command is starting up. You can use this to update service container bindings, inspect the command instance properties or whatever you wish.
```
